bin/plugin/commands/release.js

```diff
@@ -71,34 +71,64 @@ async function runReleaseBranchCreationStep(
 ) {
 	let version, releaseBranch, versionLabel;
 	await runStep( 'Creating the release branch', abortMessage, async () => {
+		let parsedMajorVersion;
+
+		const getComponentsOfRC = ( ver ) => ver.match( /(.+-rc\.)([0-9]+)/ );
 		const packageJsonPath = gitWorkingDirectoryPath + '/package.json';
 		const packageJson = readJSONFile( packageJsonPath );
-		const nextMajor = getNextMajorVersion( packageJson.version );
-		const parsedMajorVersion = semver.parse( nextMajor );
+		const rcComponents = getComponentsOfRC( packageJson.version );
+		const isRC = !! rcComponents;
+
+		// if the last tag is an RC then we can assume that we should generate a new rc with the same version number
+		if ( isRC ) {
+			const [ , current, rc ] = rcComponents;
+			const nextRC = Number( rc ) + 1;
+
+			version = `${ current }${ nextRC }`;
+			versionLabel = `${ current.split( '-' )[ 0 ] } RC${ nextRC }`;
+			parsedMajorVersion = semver.parse( packageJson.version );
```

Comment:
We can use `semver.inc()` for the version arithmetics here:

```js
version = semver.inc( packageVersion, 'prerelease' );
```

Reply:
TIL 👍🏻